backend/latest/src/plugin.ts

```diff
@@ -75,91 +71,105 @@ const plugins: BackendPlugins = {
       related_record_id: { equal_any: lines.map(({ id }) => id) },
     });
 
-    // Sqlite only
-    const sql_statement = `
+    const sql_result = getAggregatedAmc(
+      requisition.store_id,
+      lines.map(({ item_link_id }) => item_link_id)
+    );
+
+    return {
```

Comment:
Git seems a bit confused here, the only change to the rest of transform_requisition_line, is now reducing average_monthly_consumption (summing it up for item), previously summing was don in sql, but we need pre summed info